pkg/sql/schemachanger/scexec/exec_immediate_mutation.go

```diff
@@ -26,6 +27,7 @@ type immediateState struct {
 	descriptorsToDelete catalog.DescriptorIDSet
 	commentsToUpdate    []commentToUpdate
 	newDescriptors      map[descpb.ID]catalog.MutableDescriptor
+	addedNames          map[descpb.ID]descpb.NameInfo
```

Comment:
instead of adding `addedNames` field here, can we use `newDescriptors` instead? I suppose we can get same information from them and creating new descriptors normally means we need to add new names except for Functions. That being said, I'm fine with your approach, but just prefer to avoid adding new fields if possible.